qiskit_nature/second_q/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -34,7 +34,7 @@ class ExcitedStatesEigensolver(ExcitedStatesSolver):
 
     def __init__(
         self,
-        qubit_converter: QubitConverter,
+        qubit_converter: Union[QubitConverter, QubitMapper],
```

Comment:
In the future I would like to rename this argument to `qubit_mapper`. This will not be an easy process so it will be done in a separate PR once all the API is updated to support `QubitMapper` and works. I'm mentioning it now just as an FYI.
This will not be an easy undertaking but I will start discussing some plans on how to do this properly offline.